### PR TITLE
Added Makefile and version file

### PR DESCRIPTION
**Pull request by ICoxCH**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+.PHONY: dist
+dist: clean build package
+
+.PHONY: sonar
+sonar:
+	mvn sonar:sonar
+
+.PHONY: sonar-pr-analysis
+sonar-pr-analysis:
+	mvn sonar:sonar -P sonar-pr-analysis
```
Comment (markpit) [inline Makefile:52]: needs new line on end

File: version [hunk 2]
```diff
@@ -0,0 +1 @@
+0.1
```
Comment (markpit) [inline version:1]: needs new line on end

File: Makefile (hunk 1 above)
Comment (ICoxCH) [reply Makefile:52]: Is it a ch standard to have new lines at the end of files?

Comment (markpit) [reply Makefile:52]: github has flagged it with an icon and i think it can cause problems sometimes in shell scripts

**Pull request merged**